netbox_onboarding/worker.py

```diff
@@ -44,6 +45,12 @@ def onboard_device(task_id, credentials):  # pylint: disable=R0915
         try:
             if ot.ip_address:
                 onboarded_device = Device.objects.get(primary_ip4__address__net_host=ot.ip_address)
+
+            if OnboardingDevice.objects.filter(device=onboarded_device, enabled=False):
```

Comment:
Just in case we somehow end up in a scenario where there isn't an OnboardingDevice for the given Device, would it be better to invert this logic?

```python
if not OnboardingDevice.objects.filter(device=onboarded_device, enabled=True):
```

Reply:
Let's go with an "implicitly enabled" setting for now